script.skipit/resources/lib/skipit.py

```diff
@@ -67,35 +68,69 @@ def onAction(self, action):
           reset()
           skip_backward()
 
+
+def do_skip(skip_secs):
+    # We need to track the actual skip, in case we run past the end
+    totalTime = xbmc.Player().getTotalTime()
```

Comment:
Maybe consider reusing the same player instance in the method scope? You're initializing three instances for no reason

Reply:
Done. :)